##### Progress Check *13.1 Find the critical t values for the following hypothesis tests:
(a) two-tailed test, α = .05, df = 12 t = $\pm$ 2.179

(b) one-tailed test, lower tail critical, α = .01, df = 19 t = -2.539

(c) one-tailed test, upper tail critical, α = .05, df = 38 t = 1.697

(d) two-tailed test, α = .01, df = 48 $\pm$ 2.704

##### t Test

$z$ = $\dfrac{sample~mean - hypothesized~population~mean}{standard~error}$ = $\dfrac{\overline{X} - \mu_{hyp}}{\sigma_{\overline{X}}}$

##### t RATIO FOR A SINGLE POPULATION MEAN

$t$ = $\dfrac{sample~mean - hypothesized~population~mean}{estimated~standard~error}$ = $\dfrac{\overline{X} - \mu_{hyp}}{S_{\overline{X}}}$

##### Standard Error or Mean

$\sigma_\overline{X}$ = $\dfrac{\sigma}{\sqrt{n}}$

##### ESTIMATED STANDARD ERROR OF THE MEAN
$S_\overline{X}$ = $\dfrac{S}{\sqrt{n}}$

###### where $S_\overline{X}$ represents the estimated standard error of the mean; n equals the sample size; and s has been defined as S is the standard deviation
$S$ = $\sqrt{\dfrac{{SS}}{n-1}}$ = $\sqrt{\dfrac{{SS}}{df}}$

$SS$ = $\Sigma({X}-\overline{X})^2$ = $\Sigma{X}^2$ - $\dfrac{(\Sigma{X})^2}{n}$

##### Progress Check *13.2 A consumers’ group randomly samples 10 “one-pound” packages of ground beef sold by a supermarket. Calculate 
(a) the mean and 

(b) the estimated standard error of the mean for this sample, given the following weights in ounces: 16, 15, 14, 15, 14, 15, 16, 14, 14, 14.
(NOTE: Refer to Panels I and II of Table 13.1 for detailed guidance when calculating the mean and estimated standard error for this new set of data.)

In [1]:
import math
import statistics

weights = [16, 15, 14, 15, 14, 15, 16, 14, 14, 14]
n = len(weights)
mean = statistics.mean(weights)
std_dev = statistics.stdev(weights)
print(f'mean = {mean}\nstandard_deviation = {std_dev}')

squared_weights = [weight**2 for weight in weights]
sum_squared_weights = sum(squared_weights)

sum_of_weights_squared = (sum(weights))**2

SS = sum_squared_weights - (sum_of_weights_squared)/n

std_error_mean = std_dev/math.sqrt(n)
print(f'standard error of mean = {std_error_mean}')

mean = 14.7
standard_deviation = 0.8232726023485646
standard error of mean = 0.2603416558635551


$t$ = $\dfrac{sample~mean - hypothesized~population~mean}{estimated~standard~error}$ = $\dfrac{\overline{X} - \mu_{hyp}}{S_{\overline{X}}}$

##### ESTIMATED STANDARD ERROR OF THE MEAN
$S_\overline{X}$ = $\dfrac{S}{\sqrt{n}}$

###### where $S_\overline{X}$ represents the estimated standard error of the mean; n equals the sample size; and s has been defined as S is the standard deviation
$S$ = $\sqrt{\dfrac{{SS}}{n-1}}$ = $\sqrt{\dfrac{{SS}}{df}}$

$SS$ = $\Sigma({X}-\overline{X})^2$ = $\Sigma{X}^2$ - $\dfrac{(\Sigma{X})^2}{n}$

##### GAS MILEAGE INVESTIGATION
Federal law might eventually specify that new automobiles must average, for example, 45 miles per gallon (mpg) of gasoline. Because it’s impossible to test all new cars, compliance tests would be based on random samples from the entire production of each car model. If a hypothesis test indicates substandard performance, the manufacturer would be penalized, we’ll assume, $200 per car for the entire production. In these tests, the null hypothesis states that, with respect to the mandated mean of 45 mpg, nothing special is happening in the population for some car model—that is, there is no substandard performance and the population mean equals or exceeds 45 mpg.

From the manufacturer’s perspective, a type I error (a stiff penalty, even though the car complies with the standard) is very serious. Accordingly, to control the type I error, let’s use the .01 instead of the customary .05 level of significance. From the federal regulator’s perspective, a type II error (not penalizing the manufacturer even though the car fails to comply with the standard) also is serious. In practice, a sample size should be selected, as described in Section 11.11, to control the type II error, that is, to ensure a reasonable detection rate for the smallest decline (judged to be important) of the true population mean below the mandated 45 mpg. To simplify computations in the present example, however, the projected one-tailed test is based on data from a very small sample of only six randomly selected cars.

Type I error - akala mo meron pero wala

Type II error - akala mo wala, pero meron, meron, meron!!

For reasons that will become apparent, the z test must be replaced by a new hypothesis test, the t test. Spend a few minutes familiarizing yourself with the boxed summary for the gas mileage investigation, noting the considerable similarities between it and summaries of previous hypothesis tests with the z test.

###### Statistical Hypotheses
H0: $\mu \geq$ 45

H1: $\mu~{<}$ 45

In [12]:
import math
import statistics

mileage = [40,44,46,41,43,44]
# mileage = [40,44,46,41,43,44,42,47,45,40,41,39]
# mileage = [40,44,46]
X_sample = 43
X_hyp = 45

def standard_error_of_mean(mileage,X,X_hyp):
    n = len(mileage)
    mean = statistics.mean(mileage)
    std_dev = statistics.stdev(mileage)
    print(f'mean = {mean}\nstandard_deviation = {std_dev}')
    
    squared_mileage = [num**2 for num in mileage]
    sum_squared_mileage = sum(squared_mileage)
    
    sum_of_mileage_squared = (sum(mileage))**2
    
    SS = sum_squared_mileage - (sum_of_mileage_squared)/n
    
    std_error_mean = std_dev/math.sqrt(n)
    print(f'standard error of mean = {std_error_mean}')
    
    t = (X - X_hyp)/std_error_mean
    print(f't = {t}')

standard_error_of_mean(mileage,X_sample,X_hyp)

# Original sample size
# mean = 43
# standard_deviation = 2.1908902300206643
# standard error of mean = 0.8944271909999159
# t = -2.23606797749979

# Larger sample
# mean = 42.666666666666664
# standard_deviation = 2.570225788926064
# standard error of mean = 0.7419602755572906
# t = -2.6955621020246516

# Smaller sample
# mean = 43.333333333333336
# standard_deviation = 3.0550504633038935
# standard error of mean = 1.763834207376394
# t = -1.1338934190276815

mean = 43
standard_deviation = 2.1908902300206643
standard error of mean = 0.8944271909999159
t = -2.23606797749979


##### At 0.01 significance level, one-tailed test lower tail critical, looking at the Table B Critical values of t, degrees of freedom is df=n-1; df=5, t = -3.365 (the negative sign is because we are considering the lower tail

###### Statistical Hypotheses
H0: $\mu \geq$ 45

H1: $\mu~{<}$ 45

t = -2.23606797749979 is greater than t = -3.365 so we will retain the Null Hypothesis H0

Decision
Retain H0 at the .01 level of significance because t = –2.25 is less negative than –3.365.

Interpretation
The population mean gas mileage could equal the required 45 mpg or more. The manufacturer shouldn’t be penalized.

##### 13.9 In the gas mileage test described in this chapter, would you prefer a smaller or a larger sample size if you were
(a) the car manufacturer? Why?

Answer: At a lower sample size, t = -1.1338934190276815 which is greater(less negative) than –3.365

(b) a vigorous prosecutor for the federal regulatory agency? Why?

Answer: At a larger sample size, t = -2.6955621020246516 which is greater(less negative) than –3.365

##### Progress Check *13.3 The consumers’ group in Question 13.2 suspects that a supermarket makes extra money by supplying less than the specified weight of 16 ounces in its “one-pound” packages of ground beef. Given that a random sample of 10 packages yields a mean of 14.7 ounces and an estimated standard error of the mean of 0.26 ounce, use the customary step-by-step procedure to test the null hypothesis at the .05 level of significance with t.

##### Progress Check *13.2 A consumers’ group randomly samples 10 “one-pound” packages of ground beef sold by a supermarket. Calculate (a) the mean and (b) the estimated standard error of the mean for this sample, given the following weights in ounces: 16, 15, 14, 15, 14, 15, 16, 14, 14, 14.

##### t RATIO
$t$ = $\dfrac{sample~mean - hypothesized~population~mean}{estimated~standard~error}$ = $\dfrac{\overline{X} - \mu_{hyp}}{S_{\overline{X}}}$

##### ESTIMATED STANDARD ERROR OF THE MEAN
$S_\overline{X}$ = $\dfrac{S}{\sqrt{n}}$

###### where $S_\overline{X}$ represents the estimated standard error of the mean; n equals the sample size; and s has been defined as S is the standard deviation
$S$ = $\sqrt{\dfrac{{SS}}{n-1}}$ = $\sqrt{\dfrac{{SS}}{df}}$

$SS$ = $\Sigma({X}-\overline{X})^2$ = $\Sigma{X}^2$ $-$ $\dfrac{(\Sigma{X})^2}{n}$

###### Statistical Hypotheses
H0: $\mu \geq$ 16

H1: $\mu~{<}$ 16

In [3]:
import math
import statistics
weights = [16, 15, 14, 15, 14, 15, 16, 14, 14, 14]
n = 10
X_sample = 14.7
X_hyp = 16
# std_err_mean = 0.26
# significance level of 0.05

n = len(weights)
mean = statistics.mean(weights)
s = statistics.stdev(weights)
print(f'mean = {mean}\nstandard_deviation = {std_dev}')

sq_values_each_weight = [weight**2 for weight in weights]
print(sq_values_each_weight)

sum_sq_values_each_weight = sum(sq_values_each_weight)
print(sum_sq_values_each_weight)

sum_all_weights = sum(weights)
sum_all_weights_squared = sum_all_weights**2
print(sum_all_weights_squared)

SS = sum_sq_values_each_weight - (sum_all_weights_squared/n)
print(f'SS = {SS}')

std_err_mean = s/math.sqrt(n)
print(std_err_mean)

t = (X_sample - X_hyp)/std_err_mean
t

mean = 14.7
standard_deviation = 0.8232726023485646
[256, 225, 196, 225, 196, 225, 256, 196, 196, 196]
2167
21609
SS = 6.099999999999909
0.2603416558635551


-4.993438317382946

##### # According to the Table Ba CRITICAL VALUES OF t for a one-tailed test at a significance value of 0.05, degrees of freedom df = n-1 = 9, the value of t=-1.833, negative because we are considering the lower proportion of the curve according to the hypothesis statement

The acceptable value is only greater than or equal to -1.833, therefore with a computed value of -4.993438317382946, which is more negative than the acceptable value(less than -1.833 in the normal curve), we will reject the null hypothesis, and we can say that the weight of the ground beef in the market is less than the standard 16 ounces


###### Statistical Hypotheses
H0: $\mu \geq$ 16

H1: $\mu~{<}$ 16

##### CONFIDENCE INTERVALS FOR μ BASED ON t
CONFIDENCE INTERVAL FOR μ BASED ON t

$\overline{X} \pm (t_{conf})(S_{\overline{X}})$

where:

$\overline{X}$ - Sample Mean

$t_{conf}$ - represents a number (distributed with n − 1 degrees of freedom) from the t tables, which satisfies the confidence specifications for the confidence interval

$S_{\overline{X}}$ - represents the estimated standard error of the mean

##### Reference to GAS MILEAGE INVESTIGATION
At 95% confidence level, which means a p-value of 5% or 0.05 and degrees of freedom df=n-1 = 6 - 1 = 5, then looking at the left panel of Table Ba CRITICAL VALUES OF t for a two-tailed test, has a $t_{conf}$ = $\pm$ 2.571

In [4]:
import math
import statistics

mileage = [40,44,46,41,43,44]
X_sample = 43
X_hyp = 45

def standard_error_of_mean(mileage,X,X_hyp):
    n = len(mileage)
    mean = statistics.mean(mileage)
    std_dev = statistics.stdev(mileage)
    # print(f'mean = {mean}\nstandard_deviation = {std_dev}')
    
    squared_mileage = [num**2 for num in mileage]
    sum_squared_mileage = sum(squared_mileage)
    
    sum_of_mileage_squared = (sum(mileage))**2
    
    SS = sum_squared_mileage - (sum_of_mileage_squared)/n
    
    std_error_mean1 = std_dev/math.sqrt(n)

    return std_error_mean1

In [5]:
# Substituting the values to the formula:
tconf = 2.571
CI1 = X_sample - (tconf*standard_error_of_mean(mileage,X_sample,X_hyp))
CI2 = X_sample + (tconf*standard_error_of_mean(mileage,X_sample,X_hyp))
print(f'It can be claimed, with 95 percent confidence, that the interval between {CI1} and {CI2} includes the true mean gas mileage for all of the cars in the population.')

It can be claimed, with 95 percent confidence, that the interval between 40.70042769193922 and 45.29957230806078 includes the true mean gas mileage for all of the cars in the population.


##### Progress Check *13.4 The consumers’ group (in Question 13.3) concludes that, in spite of the claims of the supermarket, the mean weight of its “one-pound” packages of ground beef drops below the specified 16 ounces even when chance sampling variability is taken into account.

##### Progress Check *13.3 The consumers’ group in Question 13.2 suspects that a supermarket makes extra money by supplying less than the specified weight of 16 ounces in its “one-pound” packages of ground beef. Given that a random sample of 10 packages yields a mean of 14.7 ounces and an estimated standard error of the mean of 0.26 ounce, use the customary step-by-step procedure to test the null hypothesis at the .05 level of significance with t.

(a) Construct a 95 percent confidence interval for the true weight of all “one-pound” packages of ground beef.

(b) Interpret this confidence interval.

Confidence level at 95% which means a p-value of 0.05, which means a two-tailed test because we are considering both tails of the normal curve, $t_{conf}$ = $\pm$2.262 for df=n-1 = 5

##### ESTIMATED STANDARD ERROR OF THE MEAN
$S_\overline{X}$ = $\dfrac{S}{\sqrt{n}}$

###### where $S_\overline{X}$ represents the estimated standard error of the mean; n equals the sample size; and s has been defined as S is the standard deviation
$S$ = $\sqrt{\dfrac{{SS}}{n-1}}$ = $\sqrt{\dfrac{{SS}}{df}}$

$SS$ = $\Sigma({X}-\overline{X})^2$ = $\Sigma{X}^2$ $-$ $\dfrac{(\Sigma{X})^2}{n}$

CONFIDENCE INTERVAL FOR μ BASED ON t

$\overline{X} \pm (t_{conf})(S_{\overline{X}})$

In [6]:
import math
import statistics
weights = [16, 15, 14, 15, 14, 15, 16, 14, 14, 14]
X_sample = 14.7
tconf = 2.262
s = statistics.stdev(weights)

square_of_each_weight_values = [weight**2 for weight in weights]
sum_of_square_of_each_weight_values = sum(square_of_each_weight_values)

sum_of_weight_values = sum(weights)
square_of_sum_of_weight_values = sum_of_weight_values**2

std_err_mean = s/math.sqrt(n)
std_err_mean

CI1 = X_sample - (tconf*std_err_mean)
CI2 = X_sample + (tconf*std_err_mean)
print(f'We can be 95 percent confident that the interval between {CI1} and {CI2} ounces includes the true population mean weight for all packages.')

# We can be 95 percent confident that the interval between 14.11 and 15.29 ounces includes the true population mean weight for all packages.

We can be 95 percent confident that the interval between 14.111107174436638 and 15.28889282556336 ounces includes the true population mean weight for all packages.


##### ASSUMPTIONS
Whether testing hypotheses or constructing confidence intervals for population means,$use~t~rather~than~z~whenever,~as~almost~always~is~the~case,~the~population~standard~deviation~is~unknown.$ Strictly speaking, when using t, you must assume that the underlying population is normally distributed. Even if this normality assumption is violated, t retains much of its accuracy as long as the sample size isn’t too small. If a very small sample (less than about 10) is being used and you believe that the sample originates from a non-normal population—possibly because of a pronounced positive or negative skew among the observations in the sample—it would be wise to increase the size of the sample before testing a hypothesis or constructing a confidence interval.

##### 13.5 A library system lends books for periods of 21 days. This policy is being reevaluated in view of a possible new loan period that could be either longer or shorter than 21 days. To aid in making this decision, book-lending records were consulted to determine the loan periods actually used by the patrons. A random sample of eight records revealed the following loan periods in days: 21, 15, 12, 24, 20, 21, 13, and 16. Test the null hypothesis with t, using the .05 level of significance.

###### Statistical Hypotheses
H0: $\mu =$ 21

H1: $\mu~!=$ 21

##### ESTIMATED STANDARD ERROR OF THE MEAN
$S_\overline{X}$ = $\dfrac{S}{\sqrt{n}}$

###### where $S_\overline{X}$ represents the estimated standard error of the mean; n equals the sample size; and s has been defined as S is the standard deviation
$S$ = $\sqrt{\dfrac{{SS}}{n-1}}$ = $\sqrt{\dfrac{{SS}}{df}}$

$SS$ = $\Sigma({X}-\overline{X})^2$ = $\Sigma{X}^2$ $-$ $\dfrac{(\Sigma{X})^2}{n}$

CONFIDENCE INTERVAL FOR μ BASED ON t

$\overline{X} \pm (t_{conf})(S_{\overline{X}})$

##### t RATIO
$t$ = $\dfrac{sample~mean - hypothesized~population~mean}{estimated~standard~error}$ = $\dfrac{\overline{X} - \mu_{hyp}}{S_{\overline{X}}}$

In [7]:
import math
import statistics

days = [21, 15, 12, 24, 20, 21, 13, 16]
n = len(days)
df = n - 1
X_hyp = 21
tconf = 2.365
# Confidence level of 95%, 0.05 significance level, two-tailed test, longer or shorter than 21 days, at df=n-1=7, the t value is 2.365

mean = statistics.mean(days)
print(f'mean = {mean}')

std_dev = statistics.stdev(days)
print(f'standard deviation = {std_dev}')

each_day_squared = [day**2 for day in days]
# print(each_day_squared)

total_each_day_squared = sum(each_day_squared)
# print(total_each_day_squared)

total_days = sum(days)
# print(total_days)

squared_total_days = total_days**2
# print(squared_total_days)

SS = total_each_day_squared - (squared_total_days/n)
# print(SS)

S = math.sqrt(SS/df)
# print(S)

# print(S == std_dev)

std_err_mean = S/math.sqrt(n)
print(std_err_mean)

# confidence intervals
CI1 = X_sample - (2.365*std_err_mean)
CI2 = X_sample + (2.365*std_err_mean)
print(CI1,CI2)

t = (mean - X_hyp)/std_err_mean
print(t)

mean = 17.75
standard deviation = 4.334248987508019
1.532388425208924
11.075901374380894 18.324098625619104
-2.120872193064822


##### Decision
Retain H0 at the .05 level of significance because t = –2.12 is less negative
than –2.365.

Interpretation: No evidence that, on average, library patrons borrow books for longer or shorter
periods than 21 days.

##### 13.6 It’s well established, we’ll assume, that lab rats require an average of 32 trials in a complex water maze before reaching a learning criterion of three consecutive error-less trials. To determine whether a mildly adverse stimulus has any effect on performance, a sample of seven lab rats were given a mild electrical shock just before each trial.

(a) Given that X = 34.89 and s = 3.02, test the null hypothesis with t, using the .05 level of significance.

(b) Construct a 95 percent confidence interval for the true number of trials required to learn the water maze.

(c) Interpret this confidence interval.

###### Statistical Hypotheses
H0: $\mu \leq$ 32 trials

H1: $\mu >$ 32 trials

##### ESTIMATED STANDARD ERROR OF THE MEAN
$S_\overline{X}$ = $\dfrac{S}{\sqrt{n}}$

###### where $S_\overline{X}$ represents the estimated standard error of the mean; n equals the sample size; and s has been defined as S is the standard deviation
$S$ = $\sqrt{\dfrac{{SS}}{n-1}}$ = $\sqrt{\dfrac{{SS}}{df}}$

$SS$ = $\Sigma({X}-\overline{X})^2$ = $\Sigma{X}^2$ $-$ $\dfrac{(\Sigma{X})^2}{n}$

CONFIDENCE INTERVAL FOR μ BASED ON t

$\overline{X} \pm (t_{conf})(S_{\overline{X}})$

##### t RATIO
$t$ = $\dfrac{sample~mean - hypothesized~population~mean}{estimated~standard~error}$ = $\dfrac{\overline{X} - \mu_{hyp}}{S_{\overline{X}}}$

In [8]:
# a) 95% confidence level at a significance level/p-value of 0.05, one-tailed test upper tail critical, df=n-1=6
#  t-value = +1.943
import math
import statistics

n = 7
df = n-1
s = 3.02
X = 34.89
X_hyp = 32

std_err_mean = s/math.sqrt(n)
print(f'std_err_mean = {std_err_mean}')

t_ratio = (X-X_hyp)/std_err_mean
print(f't_ratio = {t_ratio}')
print('t_value = 1.943')

CI1 = X - (1.943*std_err_mean)
CI2 = X + (1.943*std_err_mean)
print(CI1,CI2)

std_err_mean = 1.1414527084878663
t_ratio = 2.531861353965784
t_value = 1.943
32.67215738740808 37.107842612591924


The computed t_ratio = 2.531861353965784 is larger than the t_value = 1.943, therefore we will reject the Null Hypothesis.
The confidence intervals are 32.67215738740808 to 37.107842612591924, and therefore we are 95% confident that the true number of trial days is between these two limits, which is greater than the hypothesized value of 32 days, which led to the decision to reject the null hypothesis.

##### 13.7 Is the temperature of the earth getting warmer because heat is trapped by greenhouse gas emissions, such as carbon dioxide, in the earth’s atmosphere? The National Climatic Data Center reports on its website at http://www.ncdc.noaa.gov/pub/data/anomalies.html that the average global temperatures for recent years have deviated above the long term mean temperature for the entire twentieth century. Expressed in Fahrenheit degrees, the annual deviations above the long term mean temperature for each of ten recent years, listed in chronological order up to 2015, were 1.15, 1.15, 1.01, 1.03 1.15, 1.22, 1.03, 1.13, 1.21, and 1.35.

(a) Given that X = 1.14 and s = 0.10 for these ten years, use t at the .01 level of significance to test the null hypothesis that the temperature of earth is not getting warmer. In other words, could the sample mean deviation for these ten years have originated from a population of annual deviations for the entire twentieth century having a mean deviation equal to zero?

(b) If appropriate (because the null hypothesis has been rejected), construct a 99 percent confidence interval and interpret this interval.

###### Statistical Hypotheses
H0: $\mu \leq$ 0

H1: $\mu >$ 0

##### ESTIMATED STANDARD ERROR OF THE MEAN
$S_\overline{X}$ = $\dfrac{S}{\sqrt{n}}$

###### where $S_\overline{X}$ represents the estimated standard error of the mean; n equals the sample size; and s has been defined as S is the standard deviation
$S$ = $\sqrt{\dfrac{{SS}}{n-1}}$ = $\sqrt{\dfrac{{SS}}{df}}$

$SS$ = $\Sigma({X}-\overline{X})^2$ = $\Sigma{X}^2$ $-$ $\dfrac{(\Sigma{X})^2}{n}$

CONFIDENCE INTERVAL FOR μ BASED ON t

$\overline{X} \pm (t_{conf})(S_{\overline{X}})$

##### t RATIO
$t$ = $\dfrac{sample~mean - hypothesized~population~mean}{estimated~standard~error}$ = $\dfrac{\overline{X} - \mu_{hyp}}{S_{\overline{X}}}$

In [9]:
import statistics
import math

temps = [1.15, 1.15, 1.01, 1.03, 1.15, 1.22, 1.03, 1.13, 1.21, 1.35]
n = len(temps)
df = n-1
s = 0.10
X = 1.14

# a)
# At a confidence level of 99% and a significance level/p-value of 0.01 for a one-tailed test upper tail critical, df=n-1=9 t_value = 2.821
mean = statistics.mean(temps)
print(f'mean = {mean}')

std_dev = statistics.stdev(temps)
print(f'standard deviation = {std_dev}')

std_err_mean = s/math.sqrt(n)
print(f'std_err_mean = {std_err_mean}')

t_ratio = (1.14-0)/std_err_mean
print(f't_ratio = {t_ratio}')
# Decision Reject H0 at the .01 level of significance because the t of 36.04996532591952 exceeds the critical t of 2.821.

mean = 1.143
standard deviation = 0.10371435130523966
std_err_mean = 0.03162277660168379
t_ratio = 36.04996532591952


In [10]:
# b) Since H0 was rejected, a confidence interval is appropriate. At 99% confidence, significance level/p-value of 0.01
# we will do a two-tailed test, df=n-1=9, that has a t_value = 3.25
# 1.05079214720665 1.2292078527933499

CI1 = X - (3.25*std_err_mean)
CI2 = X + (3.25*std_err_mean)
print(CI1,CI2)

1.0372259760445275 1.2427740239554723


##### We can be 99 percent confident that the interval between 1.0372259760445275 and 1.2427740239554723 degrees Fahrenheit includes the true mean increase in global temperature above the average temperature for the twentieth century.

##### 13.8 Assume that, on average, healthy young adults dream 90 minutes each night, as inferred from a number of measures, including rapid eye movement (REM) sleep. An investigator wishes to determine whether drinking coffee just before going to sleep affects the amount of dream time. After drinking a standard amount of coffee, dream time is monitored for each of 28 healthy young adults in a random sample. Results show a sample mean, X , of 88 minutes and a sample standard deviation, s, of 9 minutes.
(a) Use t to test the null hypothesis at the .05 level of significance.

(b) If appropriate (because the null hypothesis has been rejected), construct a 95 percent confidence interval and interpret this interval.

##### ESTIMATED STANDARD ERROR OF THE MEAN
$S_\overline{X}$ = $\dfrac{S}{\sqrt{n}}$

###### where $S_\overline{X}$ represents the estimated standard error of the mean; n equals the sample size; and s has been defined as S is the standard deviation
$S$ = $\sqrt{\dfrac{{SS}}{n-1}}$ = $\sqrt{\dfrac{{SS}}{df}}$

$SS$ = $\Sigma({X}-\overline{X})^2$ = $\Sigma{X}^2$ $-$ $\dfrac{(\Sigma{X})^2}{n}$

CONFIDENCE INTERVAL FOR μ BASED ON t

$\overline{X} \pm (t_{conf})(S_{\overline{X}})$

##### t RATIO
$t$ = $\dfrac{sample~mean - hypothesized~population~mean}{estimated~standard~error}$ = $\dfrac{\overline{X} - \mu_{hyp}}{S_{\overline{X}}}$

###### Statistical Hypotheses
H0: $\mu =$ 90

H1: $\mu \neq$ 90

In [11]:
import statistics
import math

n = 28
X = 88
s = 9
X_hyp = 90

# a) 95% confidence with a significance level/p-value = 0.05, df = n-1=27, two-tailed test, t_value = 2.052
std_err_mean = s/math.sqrt(n)
print(f'std_err_mean = {std_err_mean}')

t_ratio = (X - X_hyp)/std_err_mean
print(f't_ratio = {t_ratio}')
# retain null hypothesis because we have not disproven that the mean is not equal to 90 minutes

# b) Confidence interval with 95% confidence and a significance level/p-value of 0.05, two-tailed test, df = n-1 = 27, has a t_value = 2.052
CI1 = X - (2.052*std_err_mean)
CI2 = X + (2.052*std_err_mean)
print(CI1,CI2)

std_err_mean = 1.7008401285415224
t_ratio = -1.1758894715842625
84.5098760562328 91.4901239437672
